crates/jwk/src/multicodec.rs

```diff
@@ -184,3 +203,22 @@ pub enum ToMulticodecError {
     #[error("invalid input key: {0}")]
     InvalidInputKey(Error),
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_multicodec_jwk_jcs_pub() {
+        let jwk = JWK::generate_p256();
```

Comment:
This won't work without the `secp256r1` feature enabled. You can feature-gate the test.

Reply:
@timothee-haudebourg - sorry about that.  This is my first foray into Rust, can you tell?
Hopefully this fixes everything.